### ISSUE TITLE
Warning: Failed prop type: The prop `"aria-label"` is marked as required in `Grid`, but its value is `undefined` after update from v9.21.2 to 9.22.0

### ISSUE THREAD

**Issue by barbalex** [bvaughn/react-virtualized#1577]
## Bug Report

```
Warning: Failed prop type: The prop `"aria-label"` is marked as required in `Grid`, but its value is `undefined`.
    in Grid (created by List)
    in List (created by Styled(List))
    in Styled(List) (created by AutoSizer)
    in div (created by AutoSizer)
    in AutoSizer (created by Geschaefte)
    in ErrorBoundary (created by MyErrorBoundary)
    in MyErrorBoundary (created by Geschaefte)
    in div (created by styled.div)
    in styled.div (created by Geschaefte)
    in div (created by styled.div)
    in styled.div (created by Geschaefte)
    in div (created by styled.div)
    in styled.div (created by Geschaefte)
    in ErrorBoundary (created by MyErrorBoundary)
    in MyErrorBoundary (created by Geschaefte)
    in Geschaefte (created by GeschaefteLayout)
    in div (created by Pane)
    in Pane (created by SplitPane)
    in div (created by SplitPane)
    in SplitPane (created by Styled(SplitPane))
    in Styled(SplitPane) (created by GeschaefteLayout)
    in GeschaefteLayout (created by App)
    in App
printWarning @ checkPropTypes.js:20
checkPropTypes @ checkPropTypes.js:82
validatePropTypes @ react.development.js:1714
createElementWithValidation @ react.development.js:1813
render @ List.js:228
finishClassComponent @ react-dom.development.js:17160
updateClassComponent @ react-dom.development.js:17110
beginWork @ react-dom.development.js:18620
beginWork$1 @ react-dom.development.js:23179
performUnitOfWork @ react-dom.development.js:22157
workLoopSync @ react-dom.development.js:22130
performSyncWorkOnRoot @ react-dom.development.js:21756
scheduleUpdateOnFiber @ react-dom.development.js:21188
updateContainer @ react-dom.development.js:24373
(anonymous) @ react-dom.development.js:24758
unbatchedUpdates @ react-dom.development.js:21903
legacyRenderSubtreeIntoContainer @ react-dom.development.js:24757
render @ react-dom.development.js:24840
_callee$ @ index.js:31
u @ exceljs.min.js:3
(anonymous) @ exceljs.min.js:3
forEach.e.<computed> @ exceljs.min.js:3
asyncGeneratorStep @ index.js:193350
_next @ index.js:193352
Promise.then (async)
asyncGeneratorStep @ index.js:193350
_next @ index.js:193352
(anonymous) @ index.js:193352
(anonymous) @ index.js:193352
run @ index.js:13
./src/index.js @ index.js:39
__webpack_require__ @ bootstrap:789
fn @ bootstrap:100
0 @ index.js:196538
__webpack_require__ @ bootstrap:789
(anonymous) @ bootstrap:856
(anonymous) @ bootstrap:856

```

See Sandbox: https://codesandbox.io/s/react-virtualized-list-with-width-100-6wbkh

### What is the current behavior?

Above warning occurs.

### What is the expected behavior?

No warning, as in v9.21.2

### Which versions of React and react-virtualized, and which browser / OS are affected by this issue? Did this work in previous versions of react-virtualized?
|                   |          |
|-------------------|----------|
| Browser           |     chrome up to date     |
| OS                |     windows 10     |
| React             |     16.13.1     |
| React DOM         |     16.13.1     |
| react-virtualized |    9.22.0      |

**Comment by rshaul** [comment]
Also seeing this with the MultiGrid

**Comment by Hypnosphi** [comment]
Related: https://github.com/brigand/babel-plugin-flow-react-proptypes/issues/219

should be fixed by updating `babel-plugin-flow-react-proptypes` to `26.0.0`

**Comment by while-rising-ltd** [comment]
@wuweiweiwu I don't have babel-plugin-flow-react-proptypes as an enabled babel plugin and i get this very same issue. Can this be investigated further?

**Comment by Hypnosphi** [comment]
@while-rising-ltd 

>  I don't have babel-plugin-flow-react-proptypes

`react-virtualized` itself does use it, that's the problem

**Comment by chidz-1** [comment]
> should be fixed by updating `babel-plugin-flow-react-proptypes` to `26.0.0`

Got you - thought you were imply that anyone who sees that should update to that themselves.

I'll try 9.22.1, thanks for the clarification

**Comment by while-rising-ltd** [comment]
> should be fixed by updating `babel-plugin-flow-react-proptypes` to `26.0.0`

Got you now... thought you were imply that anyone who sees that should update to that themselves.

I'll try 9.22.1, thanks for the clarification
